exampleapp/App.jsx

```diff
@@ -49,6 +65,23 @@ var App = React.createClass({
               <label>leafSpacing:</label>
               <input type="range" min="0" max="20" step="1" value={this.state.leafSpacing} onChange={this.changeLeafSpacing}/>
             </fieldset>
+
+            <h2>Change some filters:</h2>
+            {
```

Comment:
nicer to make this a generator function and calling it as `{ this.generateCheckboxes() }` or something similarly named